### DISCUSSION TITLE
Locking Spacer Issue - any solution yet?

### DISCUSSION THREAD

**Discussion by DraxStorm** [aderusha/DDD-Printable-Wall-Control-System#19] [Q&A] [answered]
I've been having an issue with the locking spacers, from what I can tell there are 2 different orientations of Slots to holes and that's where the issue comes in.
The first I'll call a midslot hole - Attached Pic.
The second I'll call the top/bottom slot hole - Attached Pic
From what I can see at present all the locks are designed for the MidSlot hole, so when I use the wide slot/pegboard none of the holes line up, so was wondering if anyone has built the controls to handle the other ones yet?

![MidSlot_Hole](https://user-images.githubusercontent.com/78496937/141660591-8c998094-1c7f-4752-b7f6-2ab0e5308b1f.jpg)
.
![TopBottomSlot_Hole](https://user-images.githubusercontent.com/78496937/141660593-19c99304-df80-45ae-ba99-bd12b9bf66b2.jpg)

**Comment by aderusha** [comment, marked as answer]
Hey @DraxStorm I got something for ya - I've just pushed a bunch of models for the locking spacers which support the horizontal Wall Control panels.  Check out the centerpieces [here](https://github.com/aderusha/DDD-Printable-Wall-Control-System/tree/main/Centerpieces/Locking_spacer_for_horizontal_Wall_Control) and the locking retainers [here](https://github.com/aderusha/DDD-Printable-Wall-Control-System/tree/main/Sidepieces/Retainers).

As you noted, the vertical and horizontal Wall Control panels have different hole patterns.  This should only impact the locking mechanism, so I've gone and created compatible models for centerpieces and retainers to fit the pattern used by the horizontal panels.

**Reply by DraxStorm** [reply]
BTW, thank you, these look great.  Just sent in a few more things I've put together lately as well.

**Reply by aderusha** [reply]
Hell yeah my man, thanks for the models!
